tools/congestion-model/src/model/queue.rs

```diff
@@ -4,12 +4,13 @@ use std::collections::VecDeque;
 
 pub struct Queue {
     shard: ShardId,
+    name: String,
     messages: VecDeque<Receipt>,
 }
 
 impl Queue {
-    pub fn new(shard: ShardId) -> Self {
-        Self { shard, messages: VecDeque::new() }
+    pub fn new(shard: ShardId, name: &str) -> Self {
```

Comment:
This is needed because the queue ids are not particularly human friendly. It however puts extra work for the strategy developers because they'd need to name their queues. It also means that unless the developers come up with consistent naming the queus stats and dashboard panels won't be reusable. I'm open for better suggestions or ideas.